DilemmaGX/Animator.js

```diff
@@ -0,0 +1,773 @@
+                        value: 'easeInOutQuint'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInElastic'),
+                        value: 'easeInElastic'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeOutElastic'),
+                        value: 'easeOutElastic'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInOutElastic'),
+                        value: 'easeInOutElastic'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInBack'),
+                        value: 'easeInBack'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeOutBack'),
+                        value: 'easeOutBack'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInOutBack'),
+                        value: 'easeInOutBack'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInBounce'),
+                        value: 'easeInBounce'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeOutBounce'),
+                        value: 'easeOutBounce'
+                    },
+                    {
+                        text: this.formatMessage('Ani.easeInOutBounce'),
+                        value: 'easeInOutBounce'
+                    }
+                ],
+                Effect: [
+                    {
+                        text: this.formatMessage('Ani.color'),
+                        value: 'color'
+                    },
+                    {
+                        text: this.formatMessage('Ani.fisheye'),
+                        value: 'fisheye'
+                    },
+                    {
+                        text: this.formatMessage('Ani.whirl'),
+                        value: 'whirl'
+                    },
+                    {
+                        text: this.formatMessage('Ani.pixelate'),
+                        value: 'pixelate'
+                    },
+                    {
+                        text: this.formatMessage('Ani.mosaic'),
+                        value: 'mosaic'
+                    },
+                    {
+                        text: this.formatMessage('Ani.brightness'),
+                        value: 'brightness'
+                    },
+                    {
+                        text: this.formatMessage('Ani.ghost'),
+                        value: 'ghost'
+                    },
+                ]
+            }
+        }
+    }
+    //Defines start here
+
+    docs() {
+        console.log("Opening Animator.js documentation in new window...");
+        window.open("https://gandi.rth1.app/Docs/Animator.js.html");
```

Comment:
i am pretty sure he docs were supposed to be for the users, not for other devs. 😅

Reply:
Yeah but I didn't complete the real doc yet so I replaced it with developer's doc (because it's not testing with other ext developers)

Reply:
you should make the doc after the extension is on this repo